Remove internal build of TBB. Use standard tbb instead of tbb_preview.
Author: micketeer@gmail.com

diff -ru usher-0.4.1.orig/CMakeLists.txt usher-0.4.1/CMakeLists.txt
--- usher-0.4.1.orig/CMakeLists.txt	2021-08-12 12:23:07.248102942 +0200
+++ usher-0.4.1/CMakeLists.txt	2021-08-12 12:23:26.353054422 +0200
@@ -35,9 +35,7 @@
 
 include_directories(${Protobuf_INCLUDE_DIRS})
 
-include(${TBB_DIR}/cmake/TBBBuild.cmake)
-tbb_build(TBB_ROOT ${TBB_DIR} CONFIG_DIR TBB_DIR MAKE_ARGS tbb_cpf=1)
-find_package(TBB REQUIRED tbbmalloc tbbmalloc_proxy tbb_preview)
+find_package(TBB REQUIRED tbbmalloc tbbmalloc_proxy tbb)
 
 FIND_PACKAGE(Boost COMPONENTS program_options iostreams filesystem date_time REQUIRED)
 
